frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -75,7 +76,31 @@ if (Cypress.env('BRIDGE_AWS')) {
       listPage.rows.shouldNotExist(`${snapshotName}dup`);
     });
 
+    it('Restore a Snapshot to create a new claim from it', () => {
+      cy.clickNavLink(['Volume Snapshots']);
+      listPage.rows.clickKebabAction(snapshotName, 'Restore as new PVC');
+      modal.shouldBeOpened();
+      cy.byTestID('pvc-name').should('have.value', `${snapshotName}-restore`);
+      cy.get(SnapshotDetails.scDropdown).click();
+      cy.get(dropdownFirstItem)
+        .eq(1)
+        .click();
+      modal.submit();
+      modal.shouldBeClosed();
+      cy.exec(
+        `oc patch Deployment ${
+          testerDeployment.metadata.name
+        } --type='json' -n ${testName} -p '[${JSON.stringify(PatchForVolume)}]'`,
+      )
+        .its('stdout')
+        .then((res) => {
+          expect(res).toBe(`deployment.apps/${testerDeployment.metadata.name} patched`);
```

Comment:
We shouldn't check this text. If the command is successful (return code 0), that is sufficient to know it worked.